modules/rostests/apitests/gdi32/GetObject.c

```diff
@@ -344,6 +344,21 @@ Test_Brush(void)
 
     ok(GetObjectW((HANDLE)GDI_OBJECT_TYPE_BRUSH, sizeof(LOGBRUSH), &logbrush) == 0, "\n");
     ok(GetLastError() == ERROR_INVALID_PARAMETER, "expected ERROR_INVALID_PARAMETER, got %ld\n", GetLastError());
+
+    /* Test handle fixup */
+    hBrush = (HBRUSH)((ULONG_PTR)GetStockObject(WHITE_BRUSH) & 0xFFFF);
```

Comment:
Maybe a comment to "confirm" that it is expected that the stock object result holds in the first 16 bits? (unless this is a "known" fact?)